packages/cadl-python/test/generated/collections-basic/cadl/testserver/collectionpropertiesbasic/_operations/_operations.py

```diff
@@ -188,35 +174,20 @@ def send_collection_model(  # pylint: disable=inconsistent-return-statements
             return cls(pipeline_response, None, {})
 
     @distributed_trace
-    def get_collection_model(self, **kwargs: Any) -> JSON:
+    def get_collection_model(self, **kwargs: Any) -> _models.OutputModel:
         """get_collection_model.
 
-        :return: JSON object
-        :rtype: JSON
+        :return: OutputModel. This object is MutableMapping-compatible.
```

Comment:
nit: "This object is compatible with MutableMapping"

Reply:
done